### DISCUSSION TITLE
Request to move `asdf-odo` to this community organization

### DISCUSSION THREAD

**Discussion by rm3l** [asdf-community/.github#43] [General]
Hi there,

I would like to move [`rm3l/asdf-odo`](https://github.com/rm3l/asdf-odo) to this community org.
[`odo`](https://odo.dev) is a Developer-focused CLI for fast & iterative application development on Kubernetes and OpenShift. 

Thanks.

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)

**Reply by rm3l** [reply]
Thanks @smorimoto ! No problem - I can take a look.

**Reply by smorimoto** [reply]
All done!
